components/ui/icons/index.js

```diff
@@ -0,0 +1,15 @@
+import withSvg from '../../hoc/withSvg';
+
+import LevelLogoIcon from './LevelLogo.svg';
```

Comment:
We may want to do this like `LevelLogoSvg` and export the icon as `LevelLogoIcon`